terraform/oke/version.auto.tfvars

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2023, 2024, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# TODO: This must be incremented with every release of stack to OCI RMS template
+template_version = "0007"
```

Comment:
The template version is only to identity the version of template in a particular production region. This version does not control the version of the template to be used by the stack.